fenix/app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -59,6 +57,20 @@ import org.mozilla.fenix.theme.FirefoxTheme
 // Number of recently visited items per column.
 private const val VISITS_PER_COLUMN = 3
 
+private const val ITEM_ROW_HEIGHT = 56
```

Comment:
❓ Should we just make these `Dp` instead of `Int`? It would mean removing the `const` but in all effectiveness they would be a constant and even though it's not a lot of work, we are converting Int to Dp in every composition otherwise.